reana_client/api/client.py

```diff
@@ -1018,3 +1018,29 @@ def list_secrets(access_token):
         raise Exception(e.response.json()["message"])
     except Exception as e:
         raise e
+
+def workspaces(access_token):
+    """List available workspaces."""
+    try:
+        response, http_response = current_rs_api_client.api.workspaces(
+            access_token = access_token,
```

Comment:
`access_token` is passed, but [workspaces endpoint](https://github.com/reanahub/reana-server/pull/397/files) does not expect it. It should have [signin_required](https://github.com/reanahub/reana-server/blob/master/reana_server/rest/workflows.py#L64) decorator and [access_token](https://github.com/reanahub/reana-server/blob/master/reana_server/rest/workflows.py#L76-L81) parameter defined in openapi spec. But I see this command is placed near `ping` and `version`, if it's final place for it to be, maybe we don't need to require `access_token` for it?

Reply:
You are right about not passing the access-token. I was thinking that only the authorized users with the token can see the available workspaces, but I am not sure, do you think is better without it?